### PR TITLE
fix/ci

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

**Comment by Segelzwerg** [comment]
> partial deals with #7

actually upgraded scope of PR

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -29,13 +29,19 @@ jobs:
       fail-fast: false
       matrix:
         language: [ 'python' ]
+        python-version: [3.9]
         # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
         # Learn more:
-        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
+        # https://docs.github.com/en/free-pro-team@latest/githubmissing-module-docstring/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
```
Comment (iTitus) [inline .github/workflows/codeql-analysis.yml:35]: ???
```suggestion
        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
```
???

**Review by iTitus — CHANGES_REQUESTED** [review]
Can you change the setup-python action to v2 ?

File: requirements.txt [hunk 2]
```diff
@@ -5,3 +5,4 @@ fastapi[all]~=0.63.0
 # whist-core~=0.0.2
 pytest~=6.2.3
 pytest-cov~=2.11.1
+uvicorn~=0.13.4
```
[the remark this answers is not in the document]
Comment (Segelzwerg) [reply requirements.txt:8]: Ok I added it because it complaint locally.

[the remark this answers is not in the document]
Comment (iTitus) [reply requirements.txt]: Then you have to manually do `pip install fastapi[all]` in your venv

[the remark this answers is not in the document]
Comment (Segelzwerg) [reply requirements.txt]: Yeah I didn't realize it yesterday night. I removed uvicorn

**Review by iTitus — APPROVED** [review]

**Pull request merged**